src/views/docs/en/reference/runtime/node.md

```diff
@@ -247,3 +249,58 @@ await arc.ws.send({
   payload: { message: 'hai 🐶' }
 })
 ```
+
+### `arc.services`
```

Comment:
I'm not so sure about the `arc.services` / `await arc._loadServices()` API.

It seems like to have more symmetry between this and the other lazily-loaded runtime built-in arc API `arc.tables()`, we shouldn't expose `arc.services` as an object but instead always access it via an async method i.e. `let services = await arc.services()`. Basically combining `arc.services` and `arc._loadServices()`.

Reply:
It's definitely gotta be an async call. which can have it's result cached in the outer scope of the calling Lambda function handler.

Reply:
I've collapsed it down to a single method.